frontend/views/containers/proposals/ProposalsWidget.vue

```diff
@@ -0,0 +1,96 @@
+<template lang='pug'>
+ul.c-proposals(data-test='proposalsWidget')
+  li.c-empty(v-if='Object.keys(currentGroupState.proposals).length === 0')
+    svg-vote.c-svg
+    div
+      i18n(tag='h3') Proposals
+      i18n.c-text(tag='p') In Group Income, every member of the group gets to vote on important decisions, like removing or adding members, changing the mincome value and others.
+      i18n.has-text-1(tag='p') No one has created a proposal yet.
+  // TODO: view without current proposals
+  // TODO: button "see all proposals"
+  proposal-box(
+    v-else
+    v-for='hashes in proposals'
+    :key='hashes[0]'
+    :displayDate='false'
+    :proposalHashes='hashes'
+  )
+</template>
+
+<script>
+import SvgVote from '@svgs/vote.svg'
+import ProposalBox from '@containers/proposals/ProposalBox.vue'
+import { mapGetters } from 'vuex'
+
+export default {
+  name: 'Proposals',
+  components: {
+    ProposalBox,
+    SvgVote
+  },
+  computed: {
+    ...mapGetters([
+      'currentGroupState'
+    ]),
+    proposals () {
+      // Sort proposals by expering date (most recents at the top)
+      // TODO: Filter by only open proposals?
+      const p = this.currentGroupState.proposals
+      const sortByExpire = Object.keys(p).sort((prev, curr) => p[curr].data.expires_date_ms - p[prev].data.expires_date_ms)
+
+      // HACK/NOTE: Proposals to invite members created at the same time by
+      // the same user, should be "visually together". A solution without
+      // modifying the store/state is to group the "similiar" proposals in a sub-array.
+      // Let's create an Array of Arrays of Strings (hashes).
+      const proposalsGrouped = [] // [[hash1], [hash2], [hash3_g, hash4_g]]
+
+      sortByExpire.forEach((hash, index) => {
+        if (index === 0) {
+          proposalsGrouped.push([hash])
+          return
+        }
+
+        const current = p[hash]
+        const previous = p[sortByExpire[index - 1]]
+        const expireSameTime = current.data.expires_date_ms === previous.data.expires_date_ms
```

Comment:
Since multiple proposals can be created at the same time, and since there can potentially be a millisecond difference between their creation dates, it should check if the proposal `proposalType` and `reason` is the same instead of the proposal `expires_date_ms` (make sure to update `Invite.vue` to set the proposal reason based on what was entered by the user)

Reply:
that doesn't happen because the `expires_date_ms` is created just once right before the loop. at the moment we can't compare the reason because it's dummy, so they are all the same. and comparing a number is faster than comparing a long string.